### PR TITLE
Add section about SSH_ASKPASS

### PR DESCRIPTION
**Pull request by maralorn**
After telling my story on #general I was asked to

> t'd be cool if you were able to add this to the FAQ or Troubleshooting section on our docs https://github.com/radicle-dev/radicle-docs/

### CODE REVIEW

File: docs/using-radicle/troubleshooting.md [hunk 1]
```diff
@@ -211,7 +211,16 @@ interact with the Radicle network. For more instructions on how to do this, see
 **Note** that the `git-rad-remote` helper was included with your installation of
 **Radicle Upstream** and lives under `$HOME/.radicle/bin`.
 
+### Radicle does not prompt for password
+Every git command interacting with the rad remote prompts you for the password. It does so via git which relies on the `SSH_ASKPASS` environment variable.
+If you have that variable broken you wont get prompted for a password and see:
```
Comment (FintanH) [inline docs/using-radicle/troubleshooting.md:216]:
```suggestion
If you have that variable broken you won't get prompted for a password and see:
```

File: docs/using-radicle/troubleshooting.md [hunk 2]
```diff
@@ -211,7 +211,16 @@ interact with the Radicle network. For more instructions on how to do this, see
...
+```
+Error: Error unsealing key: Unable to decrypt secret box using the derived key
+```
+(Which is the error message for entering a wrong password also.)
```
Comment (FintanH) [inline docs/using-radicle/troubleshooting.md:220]:
```suggestion
**Note**: this is also the error message for entering the wrong password.
```

**Review by FintanH — CHANGES_REQUESTED** [review]
Thank you :pray: A couple of fixes, and could you format the text width to 80 char? :)

**Review by abbey-titcomb — APPROVED** [review]
Thanks for adding! LGTM after Fintan's changes

**Review by FintanH — APPROVED** [review]

**Pull request merged**